src/control/controlobject.h

```diff
@@ -2,190 +2,110 @@
 
 #include <QObject>
 #include <QEvent>
+#include <QSharedPointer>
+#include <functional>
 
 #include "preferences/usersettings.h"
 #include "controllers/midi/midimessage.h"
-#include "control/control.h"
+
+// Forward declarations
+class ControlDoublePrivate;
+class ConfigKey;
+
+enum class ControlFlag {
+    None = 0
+    // Future flags can be added here
+};
```

Comment:
an enum with a single member is nonsensical. It doesn't contain any information. Remove or use proiperly.